src/VisualStudio/Core/Test/ProjectSystemShim/VisualBasicCompilerOptionsTests.vb

```diff
@@ -49,6 +49,44 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim
 
         <WpfFact()>
         <Trait(Traits.Feature, Traits.Features.ProjectSystemShims)>
+        Public Sub SetCompilerOptions_LangVersion15_3()
```

Comment:
Should we also have a test for 15 or 14 or something, to make sure we're not busting the scenario where you're still using the host object?